addons/mail/static/src/components/notification_list/notification_list.js

```diff
@@ -175,7 +175,7 @@ export class NotificationList extends Component {
 }
 
 Object.assign(NotificationList, {
-    _allowedFilters: ['all', 'mailbox', 'channel', 'chat'],
+    _allowedFilters: ['all', 'mailbox', 'channel', 'chat', 'group'],
```

Comment:
I don't think we want a new tab for `group`, rather integrate `group` channels into the existing `chat` tab, similar to what is done in discuss sidebar.

Reply:
`this.update({ activeMobileNavbarTabId: thread.channel_type });` this is wrong, active tab id should not be so closely related to channel type, there should be a map between a channel type and an active tab id.

Reply:
Alright then